.circleci/config.yml

```diff
@@ -261,31 +78,74 @@ jobs:
             export PR_NUMBER=$CIRCLE_PR_NUMBER
             ./.ci/validation.sh git-check-pull-number
 
+  validation:
+    docker:
+      - image: &default-image checkstyle/jdk-11-groovy-git-mvn:11.0.13__3.0.9__2.25.1__3.6.3
```

Comment:
Just pointing this out, that this says groovy is 3.0.9 (unless I am reading this wrong), but we have specific version needed for other groovy script, 2.4.8 ( see https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#setup ) . Maybe we should consider synching so they are the same especially if we ever want to groovy in these CIs.

Reply:
Yes, long term I would like to avoid custom images altogether (other than IDEA) to make it easier to have consistent versions. Java version is not the same, either. But - by the same token, it is also not a bad idea to keep similar jobs running in different versions.

Reply:
Java makes sense.

Groovy does not as we code for specific version and if you deviate from that version by 1 bit, the code fails to compile. This is what prompted the documentation at contribution for tester. Groovy version should be consistent everywhere if we are ever to use it in more CIs.